### DISCUSSION TITLE
ARM Chip Support

### DISCUSSION THREAD

**Discussion by jdaeira** [BornToBeRoot/NETworkManager#2789] [Q&A]
Does anyone know if NETworkManager will work with the new ARM Snapdragon laptops?

**Comment by BornToBeRoot** [comment]
With Windows 11 you should be able to run x86/x64 apps on ARM. Maybe you can just test it with the portable version.

https://learn.microsoft.com/en-us/windows/arm/apps-on-arm-x86-emulation
